src/components/NumberInput/NumberInput.js

```diff
@@ -110,7 +110,7 @@ export default class NumberInput extends Component {
     disabled: false,
     hideLabel: false,
     iconDescription: 'choose a number',
-    label: ' ',
+    label: '',
```

Comment:
This looks a breaking change, do we want this change for experimental mode only...?

Reply:
@emyarod Thoughts...? Thanks!

Reply:
updated!